packages/cli/src/commands/repo/build.ts

```diff
@@ -60,29 +60,31 @@ function createScriptOptionsParser(anyCmd: Command, commandPath: string[]) {
 
     // Can't clone or copy or even use commands as prototype, so we mutate
     // the necessary members instead, and then reset them once we're done
-    const currentOpts = cmd._optionValues;
-    const currentStore = cmd._storeOptionsAsProperties;
+    const currentOpts = (cmd as any)._optionValues;
```

Comment:
logging `cmd`, it for sure looks like the internal structure is still the exact same - but i did of course have some hesitation around this file in general since it's a little bit iffy :P

Reply:
Yep ain't great x) Didn't find any other options though that kept reasonable code reuse.

If it builds is probably fine, since this is used in CI